docs/SOPs/Managed_access/Data_Transfer_SOP.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Data Transfer SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 4
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Data Transfer SOP
+Once the data contributor has notified the HCA data wrangler that they have completed the  metadata spreadsheet, the wrangler can give them access to hca-util area
+
+## Data and metadata upload prerequisites
+To upload data to hca-util contributors need credentials and their project’s area uuid. Credentials for hca-util must be zipped and password protected. 
+1. Send an email addressed just to the contributor containing the area’s uuid and instructions on how to upload files to hca-util*
+2. Send an email addressed just to the contributor containing the zipped credentials**
+3. Send an email addressed just to the contributor containing the password for the*** credentials
```

Comment:
Send an email addressed only to the contributor, containing the password for the*** credentials